### PR TITLE
#61 Phone numbers sanitization issues

### PR DESCRIPTION
**Pull request by idisblueflash**

### CODE REVIEW

File: models/point_of_interest.py [hunk 1]
```diff
@@ -51,9 +51,9 @@ def ensure_organizations_as_list(cls, organizations: Any) -> List[str]:
     def sanitize_url(cls, url: str) -> str:
         return sanitise_url(url)
 
-    @validator('phone', pre=True)
-    def sanitize_phone(cls, phone: str) -> str:
-        return sanitise_phone(phone)
+    @validator('phone')
```
Comment (littlepea) [inline models/point_of_interest.py:54]: Did you test this with spreadsheet conversion?

Comment (idisblueflash) [reply models/point_of_interest.py:54]: No. 
But we've got an existing Unit Test for it. As you can see, it makes the CircleCI broken.

By the way, I'm refactoring these parts as well.

Comment (littlepea) [reply models/point_of_interest.py:54]: Ok, please, test manually too

**Review by littlepea — APPROVED** [review]
how many phone numbers does it lose now when running spreadsheet conversion? (if any)

**Comment by idisblueflash** [comment]
4/84

They're all original empty.

**Pull request merged**